src/Microsoft.IdentityModel.Tokens/Telemetry/ITelemetryInstrumentation.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Microsoft.IdentityModel.Tokens.Telemetry
+{
+    internal interface ITelemetryInstrumentation
```

Comment:
Since this is all internal, not seeing a strong reason for this interface.
How does it help?

Reply:
It should be able to help with testing.